README.md

```diff
@@ -27,7 +27,32 @@ function mystery(n) {
     }
 }
 ```
-
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+## Answer
+T(n) = 2T($\frac{n}{3}$) + $n^2$ * n * $n^2$ + T($\frac{n}{3}$)
```

Comment:
This doesn't look right. Please start by giving the recurrence relation.

Reply:
I am trying to solve a mystery function that has three repeating sections. There are also three for loops involved: the first one is n * n, the second is n, and the third is n * n. T(n) = 3T(n/3) + n ^5. Can you provide me with some helpful hints?

Reply:
That sounds like it's going in the right direction.